tests/model/checkbox.model.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { Checkbox } from '../../src/app/core/models/checkbox.model';
+
+const EXPECTED_CHECKED_STRING = '- [x] todo';
+const EXPECTED_UNCHECKED_STRING = '- [ ] todo';
+
+describe('Checkbox', () => {
+  it('sets the correct isChecked value', () => {
```

Comment:
```suggestion
  it('.setChecked() sets the correct isChecked value', () => {
```

Missed this out! psps

Reply:
No worries and thanks 😄